.gitignore

```diff
@@ -0,0 +1,3 @@
+/vendor/
+/.idea/
+/composer.phar
```

Comment:
we don't need this in gitignore. Just remove it totally from git using `git rm composer.phar`